resources/resolver.go

```diff
@@ -736,14 +709,13 @@ func (p *Processor) Process(input interface{}) (interface{}, error) {
 	}
 }
 
-// ResolveConfig
+// ResolveResourcesFromRemoteOrLocal
```

Comment:
Why do we need such a long function name? Just call it `ResolveResources`.

We've spoken about these. :)